src/lib/services/locationService.ts

```diff
@@ -123,3 +123,29 @@ export async function setLocationCustomName(
 
   return setLocationCustomName;
 }
+
+/**
+ * Get details of specified location
+ */
+export async function getLocation(barcode: string): Promise<LocationFieldsFragment | undefined> {
+  const { location } = await stanCore.FindLocationByBarcode({ barcode });
+  if (!location) {
+    return undefined;
+  }
+  return location;
+}
+
+/**
+ * Get details of specified location
+ */
+export async function transferItems(
+  sourceBarcode: string,
+  destinationBarcode: string
+): Promise<LocationFieldsFragment | undefined> {
+  try {
+    const { transfer } = await stanCore.TransferLocationItems({ sourceBarcode, destinationBarcode });
+    return transfer;
+  } catch (e) {
+    throw e;
```

Comment:
What is the reason to catch e and then throw e? Why is that better than not catching it?